cmd/cape/cmd/key.go

```diff
@@ -0,0 +1,69 @@
+package cmd
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	"github.com/capeprivacy/cli/entities"
+	"github.com/capeprivacy/cli/sdk"
+)
+
+var keyCmd = &cobra.Command{
```

Comment:
I'm still a little unclear the purpose of this command? Is it so that they can get the key to decrypt data outside of cape if they want to?

Reply:
It's so you can encrypt data outside of Cape